src/graph/entities/node.c

```diff
@@ -13,44 +13,22 @@
 #include "../graphcontext.h"
 #include "../../query_ctx.h"
 
-Node *Node_New(const char *label) {
-	Node *n = calloc(1, sizeof(Node));
-	n->labelID = GRAPH_UNKNOWN_LABEL;
-	n->label = label;
-
+inline Node Node_New() {
+	Node n = {
+		.entity = NULL,
+		.label = NULL,
+		.labelID = GRAPH_NO_LABEL,
+	};
```

Comment:
MACRO?
```
#define NODE_NEW {.entity = NULL, .label = NULL, .labelID = GRAPH_NO_LABEL}; 
```
?

Reply:
Both of the constructors can be macros if you like! I have no preference.